src/helper/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const ALL_THEME_COMMANDS = ['new', 'init', 'sync', 'serve', 'publish', 'unpublish', 'pull', 'pull-config', 'context', 'context-list']
+export const THEME_COMMANDS = ['serve', 'sync', 'new', 'init'];
```

Comment:
for publish, unpublish, pull, pull-config commands you need to add check for environment mismatch @Shivraj0

Reply:
publish and unpublish will require context and environment check. Rest two i.e pull, pull-config won't require check for the same. @vivek8690

Reply:
It will be required for pull and pull-config as well @Shivraj0

Reply:
Done @vivek8690